### ISSUE TITLE
How can I set a release title?

### ISSUE THREAD

**Issue by Angeluz-07** [relekang/python-semantic-release#210]
I have a github project integrated to a slack channel. When I make a publish of a release(i.e `semantic-release changelog --post`) I got the following update:

![Release null](https://user-images.githubusercontent.com/17350786/78844091-5256db00-79ca-11ea-9ad0-7727ef70258c.PNG)

I found this is due to lack of release title -> https://github.com/integrations/slack/issues/574#issuecomment-542518371 . As wasn't solved in that issue I would like to know if there is any way to set the release title using this library.

**Comment by Angeluz-07** [comment]
I reviewed source code and  found that a change can be made in https://github.com/relekang/python-semantic-release/blob/bff40d53174ffe27451d82132c31b112c7bee9fd/semantic_release/hvcs.py#L117

to 
```
json={'tag_name': tag, 'name': tag, 'body': changelog, 'draft': False, 'prerelease': False}
```
I tested the change by making a publish and it worked as expected. Also I ran the tests with no problem. Could anybody help if this can be made a feature. I think it would be a good solution to the problem I described in the issue.

**Comment by AlphaMycelium** [comment]
Hi, feel free to open a pull request with your changes and I'll take a look.

**Comment by Angeluz-07** [comment]
Thanks for your help @AlphaMycelium I have opened a PR #211

**Comment by AlphaMycelium** [comment]
Closed by #211
